components/typography/TextLink.js

```diff
@@ -11,9 +12,11 @@ export default function TextLink({
     return(
         <>
         {' '}
-        <a href={href} target={target} rel="noreferrer noopener" className={classNames}>
+        <Link href={href} passHref>
```

Comment:
This Link has been added in order to provide smooth rendering of the pages using slug. If this change makes any error, do inform me, I will try to correct it or remove the changes made here.

Reply:
I think that we should check if `href` starts by the `http(s)` and if not, wrap it by `Link`. Could you propagate my suggestion?